application/classes/Ushahidi/Repository/User.php

```diff
@@ -54,9 +54,28 @@ protected function getTable()
 	// Ushahidi_Repository
 	public function getEntity(Array $data = null)
 	{
+		if (!empty($data['id']))
+		{
+			$data += [
+				'contacts' => $this->getContacts($data['id']),
+			];
+		}
 		return new User($data);
 	}
 
+	protected function getContacts($entity_id)
```

Comment:
Hopefully we can eventually just combine contacts + users so this gets less circular